lib/pwsh.rb

```diff
@@ -54,7 +54,7 @@ def self.instance(cmd, args, options = {})
       if manager.nil? || !manager.alive?
         # ignore any errors trying to tear down this unusable instance
         begin
-          manager&.exit
+          manager.exit if manager
```

Comment:
I think to properly guard this it would have to be

```suggestion
          manager.exit if defined?(manager)
```